tools/deploy/copy.sh

```diff
@@ -1,12 +1,33 @@
 #!/bin/bash -x
+SOURCE_DIR=$(readlink -f "$(dirname ${BASH_SOURCE[0]})/../..")
+BUILD_TYPE_ORDER="debug debug_asan debug_no_unity release release_asan release_coverage release_no_unity
```

Comment:
There are too many directories.
And if someone add more directories in the build.sh, we will not update `copy.sh` here.
So, do you have some suggestions?

Reply:
> There are too many directories. And if someone add more directories in the build.sh, we will not update `copy.sh` here. So, do you have some suggestions?

We can set `BUILD_TYPE_ORDER="debug release errsim rpm"`. If not in it, we choose it in alphabetical order.

Reply:
> Is it OK if we choose the directory in alphabetical order?

I think it's OK. If build path is not specified, copy.sh will output the path it choose. If it's not the correct path, we can set it explicitly.